### ISSUE TITLE
Add File Management

### ISSUE THREAD

**Issue by MauriceMEGA65** [MEGA65/mega65-user-guide#128]
Dear all,
First of all: two thumbs up and compliments to all who contributed to the manual. I've read the nov 30 edition and it looks fabulous!

As MSX and Amiga user I was always very confused on how the file management worked on the C64, and unfortunately that part is missing in the online manual of the MEGA65 (maybe it's already in draft available).

My proposal:

1. In chapter 7 - Getting started in BASIC:
--> Add instruction how to SAVE and LOAD your own created BASIC program (under BASIC 10, to the default disk)

2. In chapter 9 - C64, C65 and MEGA65 modes:
--> Create section 'File Management'.
  -> Add instructions on SAVE, LOAD, DELETE, MOVE, COPY for each of the 3 modes
       > Include the use of wildcards and syntax ("$", "*", ',8,1', etc)
       > Include the use of drive numbers (which drive is drive '8' etc)

**Comment by MauriceMEGA65** [comment]
I will add my proposal to the manual

**Comment by MauriceMEGA65** [comment]
Attached my first input. It's probably not complete and should be reviewed by someone with better C64 knowledge.
Somehow I still would like to have it in the printed User Guide, but can also understand if we want to embed it in the Book.
[File Management.pdf](https://github.com/MEGA65/mega65-user-guide/files/7446386/File.Management.pdf)

**Comment by gurcei** [comment]
Sorry for delays, just noticed you added this comment now!

Ok, looking over the .pdf, I like what it aims to do, how to do this in C65-mode versus how to do it in C64-mode. I just worry it's too big an add for now.

Maybe for now, such content could grow in alternate locations, whether it is in personal .pdf's, filehost articles, wiki-pages. And when we see the opportunity to take the cream of this crop and supplement one of our manuals, ok, let's go for it then.

**Comment by MauriceMEGA65** [comment]
Yeah, no worries. The doc is not new by the way, I’ve shared it a few times earlier on Discord.
The reason for this ticket is that we do not discribe the basics of file- and diskhandling. I’m thinking about formatting a floppy, save files to it, delete, copy.
Sure, all commands are in the BASIC section, but you need to know how they are named before you find them. Would be nice to have these commands all on one page for example.

But I do agree that it also can be on another location, outside the User Guide.
No showstopper, just something I was wondering about

**Comment by MauriceMEGA65** [comment]
Not sure what to do with this comment. I believe it's still useful for people not familiar with previous Commodore computers, but I think it needs to be addressed in the Programmers Guide or something similar.

**Comment by MauriceMEGA65** [comment]
Attached my notes about CBM file management. As a newbie in CBM-land, I made these notes to understand basic file management. Perhaps more people new with Commodore computers will be helped if these standard commands are explained in the Programmers Guide.

[File Management.pdf](https://github.com/MEGA65/mega65-user-guide/files/9758935/File.Management.pdf)

**Comment by dansanderson** [comment]
FYI: User's Guide 2nd edition has a chapter on "Using Disks." It is not a complete DOS tutorial but there is a brief introduction to common disk operations. User's Guide 2nd edition includes the BASIC reference for more detail.

I'll leave this issue open in case we want to consider a more thorough DOS tutorial in a different volume. I still have my eye on a dedicated volume for BASIC 65 where a more detailed tutorial could go.
